src/modules/accounting/accounting.py

```diff
@@ -214,21 +211,37 @@ def get_updated_modules_stats(
 
     @lru_cache(maxsize=1)
     def _get_consensus_lido_state(self, blockstamp: ReferenceBlockStamp) -> tuple[ValidatorsCount, ValidatorsBalance]:
+
         lido_validators = self.w3.lido_validators.get_lido_validators(blockstamp)
+        logger.info({'msg': 'Calculate Lido validators count', 'value': len(lido_validators)})
 
-        count = len(lido_validators)
-        total_balance = Gwei(sum(int(validator.balance) for validator in lido_validators))
+        total_balance = sum(int(validator.balance) for validator in lido_validators)
+        logger.info({'msg': 'Calculate Lido active balance (in Gwei)', 'value': total_balance})
 
-        logger.info({'msg': 'Calculate lido state on CL. (Validators count, Total balance in gwei)', 'value': (count, total_balance)})
-        return ValidatorsCount(count), ValidatorsBalance(total_balance)
+        consensus_version = self.w3.lido_contracts.accounting_oracle.get_consensus_version(blockstamp.block_hash)
+        if consensus_version > 2:
+            spec = self.w3.cc.get_config_spec()
+            if blockstamp.ref_epoch >= int(spec.ELECTRA_FORK_EPOCH):
```

Comment:
consider putting it in a separate method, like `is_electra_activated`, since it's widely used